apps/site/pages/docs/customizing.mdx

```diff
@@ -0,0 +1,89 @@
+import { SectionItem, SectionList } from 'site/components/SectionItem'
+
+<header className="hero">
+
+# Customizing
+
+**Design tokens** are parameters that affect all UI look 'n' feel and they are the **main configuration** file of a Theme. Component-specific tokens often inherit from global tokens but are named specifically to apply those tokens in component development.
+
+</header>
+
+## Global Tokens
+
+FastStore uses a default theme called `Brandless`, which provides structural styles for your components and pages. Please read [Brandless's page](/docs/brandless) to further information about our themeable architecture.
+
+Its presets are defined on `src/styles/global/tokens.scss` and it's separated in these main areas:
```

Comment:
```suggestion
Its presets are defined on `src/base/tokens.scss` and it's separated in these main areas:
```

Reply:
We are referring to ui package right?

`ui/src/base/tokens.scss`

Reply:
Nice catch!
Well, I think this is not the best way to refer to the base tokens yet. 
What do you think about adding the `tokens.scss` content at the end of the brandless section? @hellofanny @renatamottam

Reply:
WDYT about [this changes](https://github.com/vtex/faststore/pull/1636/commits/98d6f5e34f8862ff4489b0e25454a54839ddcce2)?

Reply:
Agreed!
I like this latest changes :) It's better to show the file path this way 👍